myokit/tests/test_simulation_cvodes.py

```diff
@@ -91,6 +134,13 @@ def test_no_protocol(self):
         # Check if pace was set to zero (see prop 651 / technical docs).
         self.assertTrue(np.all(d['engine.pace'] == 0.0))
 
+    def test_wrong_label_set_pacing(self):
+        # Test set_pacing with incorrect label
+        self.sim.reset()
+        self.sim.pre(50)
+        with self.assertRaisesRegex(ValueError, 'Unknown pacing label'):
+            self.sim.set_protocol(None, label='does not exist')
+
     def test_fixed_form_protocol(self):
```

Comment:
This needs to be updated to 

1. Use the new TimeSeriesProtocol type calls everywhere
2. Have a single call using the old syntax wrapped in a WarningCollector that checks that the deprecation warning is shown

Reply:
done